h/templates/app.html.jinja2

```diff
@@ -9,7 +9,7 @@
      only ever displays a single route.
      #}
     <base target="_top" href="/" />
-  {% for url in app_css_urls %}
+    {% for url in app_css_urls %}
```

Comment:
No, I think it stems from an indentation correction made during an earlier version of this PR.